### PR TITLE
remove all da & settlement related content

### PR DESCRIPTION
**Pull request by tdelabro**

### CODE REVIEW

File: starknet-rpc-test/Cargo.toml
Comment (apoorvsadana) [inline starknet-rpc-test/Cargo.toml:1]: why remove this?

**Review by apoorvsadana — APPROVED** [review]

File: starknet-rpc-test/Cargo.toml
Comment (tdelabro) [reply starknet-rpc-test/Cargo.toml:1]: The way it is implemented they require the node to be run with --da-layer --da-conf

Comment (apoorvsadana) [reply starknet-rpc-test/Cargo.toml:1]: Hmm, should we move the build state update logic to a new worker then? (in a new PR maybe)_

Comment (tdelabro) [reply starknet-rpc-test/Cargo.toml:1]: Yep probably.
We will need to do so to provide SNOS with an input.
I will create it after this one is merged

**Pull request merged**